guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -0,0 +1,233 @@
+[id="upgrading-{project-context}-or-proxy-in-place-using-leapp_{context}"]
+= Upgrading {Project} or {SmartProxy} to {EL} 9 In-Place Using Leapp
+
+Use this procedure to upgrade your {Project} or {SmartProxy} installation from {EL} 8 to {EL} 9.
+
+.Prerequisites
+* {Project} {ProjectVersion} or {SmartProxy} {ProjectVersion} running on {EL} 8.
+ifdef::satellite[]
+* Review Known Issues before you begin an upgrade.
+For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+endif::[]
+ifndef::satellite[]
+* Access to available repositories or a local mirror of repositories.
+endif::[]
```

Comment:
Ideally, list items should use parallel sentence structure (for example, if one item uses imperative voice, all should use imperative voice). I don't see a reasonable way of achieving this here, so I suggest to at least move the non-parallel item to the beginning of the list (it also looks like the most important one so it should really be mentioned first).

```suggestion
ifdef::satellite[]
* Review Known Issues before you begin an upgrade.
For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
endif::[]
* {Project} {ProjectVersion} or {SmartProxy} {ProjectVersion} running on {EL} 8.
ifndef::satellite[]
* Access to available repositories or a local mirror of repositories.
endif::[]
```

And let me add that I absolutely LOVE the idea to add reviewing known issues as a prerequisite.